package-lock.json

```diff
@@ -18,8 +18,7 @@
     "balanced-match": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.0.tgz",
-      "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c=",
-      "dev": true
+      "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c="
```

Comment:
The changes in this file occurred automatically when I ran `npm install tmp`. Unsure if they're desirable or not, please let me know if this needs correcting (e.g. I'm not sure what `"dev": true` was doing).